
---
string name

string[] subscriptions
string[] subscription_types
string[] publications
string[] publication_types
string[] service_clients
string[] service_client_types
string[] service_servers
string[] service_server_types
